objects/LuaScript.ttslua

```diff
@@ -1709,18 +1713,19 @@ function setupButtonsMisc()
       end
     end
   end
+
 end
 
--- This function will do a variety of things to setup the campaign including moving campiagn related objects to the table, determining first player, etc
+--This function will do a variety of things to setup the campaign including moving campiagn related objects to the table, determining first player, etc
 function setupCampaignMisc()
   local campaignBag = getObjectFromGUID(bags.campaignItems)
 
-  -- shuffle bonus encounter cards and reposition them to make space for the chapter 6 deck
+    -- shuffle bonus encounter cards and reposition them to make space for the chapter 6 deck
   for i = 1, 5 do
     local zone = getObjectFromGUID(deckZones["bonus" .. i])
     for _, obj in ipairs(zone.getObjects()) do
       if obj.type == "Deck" then
-        obj.setPosition(obj.getPosition() + Vector(-4, 0, 0))
+        obj.setPosition(obj.getPosition() + vector(-4.0, 0, 0))
```

Comment:
What's this about? Looks like an error since `Vector` needs to be capatilized

Reply:
I'm actually not sure...I didn't touch this piece of code as far as I remember. I'll just change it to Vector